src/shared/models/entity/Player.entity.ts

```diff
@@ -878,6 +878,14 @@ export class Player implements IPlayer {
   }
 
   public getDefaultChoice(choices: string[]): string {
+    if(this.$personalities.isActive('Fancypants')) {
+      if(includes(choices, 'Double')) return 'Double';
+      if(includes(choices, 'Sell')) return 'Sell';
+      if(includes(choices, 'Inventory')) return 'Inventory';
+      if(choices.length === 2 && includes(choices, 'Yes') && includes(choices, 'No')){
+        return Math.random() > 0.5 ? 'Yes' : 'No'
```

Comment:
if `sample` is available here, use that instead of `Math.random()` (it is biased)

Reply:
All fixed. My bad